.github/workflows/main.yml

```diff
@@ -34,7 +34,7 @@ jobs:
         run: mvn package -Pnative
         
       - name: Docker build image
-        run: docker build -f src/main/docker/Dockerfile.native -t akaka4ka/osmc-back-end .
+        run: docker build -t akaka4ka/osmc-back-end .
```

Comment:
Is ```akaka4ka/osmc-back-end``` path will be correct for any users?

Reply:
Yes, this is path to repo on Docker Hub, that I have created for back end images